openmls/src/group/mls_group/errors.rs

```diff
@@ -100,3 +92,79 @@ pub enum UnverifiedMessageError {
     #[error(transparent)]
     InvalidCommit(#[from] StageCommitError),
 }
+
+/// Create message error
+#[derive(Error, Debug, PartialEq, Clone)]
+pub enum CreateMessageError {
+    #[error(transparent)]
+    LibraryError(#[from] LibraryError),
+    #[error("The own CredentialBundle could not be found in the key store.")]
+    NoMatchingCredentialBundle,
+    #[error(transparent)]
+    GroupStateError(#[from] MlsGroupStateError),
+}
+
+/// Add members error
+#[derive(Error, Debug, PartialEq, Clone)]
+pub enum AddMembersError {
+    #[error(transparent)]
+    LibraryError(#[from] LibraryError),
+    #[error("The own CredentialBundle could not be found in the key store.")]
+    NoMatchingCredentialBundle,
+    #[error(transparent)]
+    EmptyInput(#[from] EmptyInputError),
+    #[error(transparent)]
+    CreateCommitError(#[from] CreateCommitError),
+    #[error(transparent)]
+    GroupStateError(#[from] MlsGroupStateError),
+}
+
+/// Propose add members error
+#[derive(Error, Debug, PartialEq, Clone)]
+pub enum ProposeAddMemberError {
+    #[error(transparent)]
+    LibraryError(#[from] LibraryError),
+    #[error("The own CredentialBundle could not be found in the key store.")]
+    NoMatchingCredentialBundle,
+    #[error("The new member does not support all required extensions.")]
```

Comment:
Here it could be interesting to know which extension type the offender is.

Reply:
Agreed, but there is much more missing and I filed #812 for that.